app/src/main/java/org/mifos/mobile/ui/savings_account_application/SavingsAccountApplicationFragment.kt

```diff
@@ -0,0 +1,75 @@
+package org.mifos.mobile.ui.savings_account_application
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.accounts.savings.SavingsWithAssociations
+import org.mifos.mobile.ui.activities.base.BaseActivity
+import org.mifos.mobile.ui.enums.SavingsAccountState
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import org.mifos.mobile.utils.Constants
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedParcelable
+import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedSerializable
+
+/*
+* Created by saksham on 30/June/2018
+*/
+@AndroidEntryPoint
+class SavingsAccountApplicationFragment : BaseFragment() {
```

Comment:
Awesome Job! Now later on when we will implement compose navigation we only need to pick the compose and done.